docs/en/README.md

```diff
@@ -39,35 +39,30 @@ $ yarn install
 $ yarn dev
 ```
 
+**How to disable accessibility（vue-axe)**
 
-[PLEASE TRANSLATE ME]
-
-**アクセシビリティチェック（vue-axe）を無効にする方法**
-
-- 開発用ローカルサーバが重い場合、以下のようにアクセシビリティチェックを無効にして起動することができます。
+- Disable accessibility when local server is heavy
```

Comment:
We already have the translation of these in #3076, that'll be merged soon.
I'm deeply sorry, but would you discard these changes?

Reply:
Sure. No problem at all.